tools/run_make_firmwares3.py

```diff
@@ -41,16 +41,26 @@ def findSTM32CubeIDEGnuTools(search_root):
                 st_cubeide_dir = file
                 st_cubeide_ver_nr = 0
     if st_cubeide_dir != '':
-        st_dir = os.path.join(search_root,st_cubeide_dir,'STM32CubeIDE','plugins')
+        if os.name == 'posix':
+            st_dir = os.path.join(search_root,st_cubeide_dir,'plugins')
+        else:
+            st_dir = os.path.join(search_root,st_cubeide_dir,'stm32cubeide','plugins')
 
     gnu_dir = ''
     for dirpath in os.listdir(st_dir):
-        if 'mcu.externaltools.gnu-tools-for-stm32' in dirpath and 'win32' in dirpath:
-            gnu_dir = dirpath 
+        if os.name == 'posix':
+            if 'mcu.externaltools.gnu-tools-for-stm32' in dirpath and 'linux' in dirpath:
+                gnu_dir = dirpath 
+        else:
+            if 'mcu.externaltools.gnu-tools-for-stm32' in dirpath and 'win32' in dirpath:
+                gnu_dir = dirpath 
     
     return st_dir, gnu_dir    
 
-ST_DIR,GNU_DIR = findSTM32CubeIDEGnuTools(os.path.join("C:/",'ST'))
+if os.name == 'posix':
+    ST_DIR,GNU_DIR = findSTM32CubeIDEGnuTools(os.path.join("/opt",'st'))
```

Comment:
????
that's just you or everyone?

Reply:
This is the path that the Linux version of the IDE package uses.